scripts/run_integration_tests.sh

```diff
@@ -3,39 +3,44 @@ set -x
 
```

Comment:
I actually think having two versions of the script would be neater.

Reply:
Done